wagtail/blocks/stream_block.py

```diff
@@ -304,6 +309,51 @@ def get_prep_value(self, value):
             # round-trips to the full data representation and back)
             return value.get_prep_value()
 
+    def normalize(self, value):
+        if value is None or value == "":
```

Comment:
This was copied more-or-less verbatim from `StreamField._to_python`. It probably requires some cooperation with `StreamValue` to honour the `normalize` contract in the lazy case - as is, if the input is the "list of dicts" representation, the subvalues won't be normalized.